.github/workflows/tests.yml

```diff
@@ -10,7 +10,23 @@ on:
 
 jobs:
 
+  wsi-artifacts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download WSI artifacts
+        id: load-wsis
+        run: |
+          mkdir -p tests/fixtures/external_wsi
+          wget http://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/CMU-3.svs -P tests/fixtures/external_wsi
```

Comment:
for now it's ok like this because we only have 1 slide to download, but maybe for the future we should read these URLs from a file

Reply:
+1